parquet/src/main/java/org/apache/iceberg/parquet/TypeWithSchemaVisitor.java

```diff
@@ -84,19 +84,18 @@
               }
 
               return visitor.list(list, group, elementResult);
-
             } finally {
               visitor.fieldNames.pop();
             }
 
           case MAP:
```

Comment:
Instead of suppressing, we could break these into methods by case.

Reply:
I can do that, but that will mess up the git history. It is up to you.